Sources/Testing/Running/EntryPoint.swift

```diff
@@ -260,14 +233,14 @@ func runTests(options: [Event.ConsoleOutputRecorder.Option], configuration: Conf
 
 extension [Event.ConsoleOutputRecorder.Option] {
   /// The set of options to use when writing to the standard error stream.
-  static var forStandardError: Self {
+  static func `for`(_ fileHandle: borrowing FileHandle) -> Self {
```

Comment:
Why not make this an `init`?

Reply:
Custom initializers on array types are somewhat discouraged, and it reads better this way IMHO.